src/Client/Updater.php

```diff
@@ -578,11 +578,7 @@ protected function verify(string $target, StreamInterface $data): void
     {
         $this->refresh();
 
-        // @todo Handle the possibility that the target's metadata might not be
-        // in targets.json.
-        // @see https://github.com/php-tuf/php-tuf/issues/116
-        $targetsMetadata = TargetsMetadata::createFromJson($this->durableStorage['targets.json']);
-
+        $targetsMetadata = $this->getMetadataForTarget($target);
```

Comment:
What happens if $targetsMetadata comes back NULL? Should we throw an exception?

Reply:
fixed